README.md

```diff
@@ -797,6 +797,16 @@ crud.cfg{stats_quantile_tolerated_error = 1e-4}
 ```
 See [tarantool/metrics#189](https://github.com/tarantool/metrics/issues/189) for
 details about the issue.
+You can also configure quantile `age_bucket_count` and `max_age_time`:
+```lua
+crud.cfg{
+    stats_quantile_age_bucket_count = 3,
+    stats_quantile_max_age_time = 30,
+}
```

Comment:
What are default values?

Reply:
https://github.com/tarantool/crud/blob/aca29f3bfb092f32e2996716d26a529c5b00d4c7/crud/stats/init.lua#L534-L538

Reply:
I mean, I don't see it in the readme.

Reply:
Added to README